pages/pipelines/tags.md

```diff
@@ -0,0 +1,13 @@
+# Pipeline tags
+
+Pipeline tags allow you to tag and search for your pipelines using the search bar. Tags are beneficial when you have many pipelines and would like to group and filter through them quickly.
+
+<%= image "pipeline-tag-search.png", alt: "Imagine showing pipeline tag search" %>
+
+## Using tags
+
+Tags are unique per pipeline. You can only add the same tag once. Tags are limited to five per pipeline, and are no longer than 64 chars. Tag names support emoji and text.
+
+<%= image "pipeline-tag-settings.png", alt: "Imagine showing pipeline tag settings" %>
+
+Tags are available through Pipeline Settings and the APIs; you can set it through the tags property in Pipelines Rest API or under pipeline types through GraphQL API.
```

Comment:
Just some more general suggestions. In particular, I try to avoid semi-colons because they're often a sign that the sentence is too complex. I've also included some links to the API docs, so please check they're right.

```suggestion
You can tag a pipeline by navigating to its _Pipeline Settings_ or using the API. In REST, use the `tags` property on the [Pipeline REST API](/docs/apis/rest-api/pipelines). In GraphQL, use the `tag` field on the [`pipelineUpdate` mutation](/docs/apis/graphql/schemas/mutation/pipelineupdate).

To use the same tag across multiple pipelines, you must create the same tag on each pipeline.
```

Is the last sentence I added right? Since the value is about filtering/grouping, I thought it would be good to provide guidance on how someone sets that up.